test/metabase/models/card_test.clj

```diff
@@ -775,3 +776,23 @@
                         Dashboard
                         before
                         changes)))))))))
+
+(deftest storing-metabase-version
+  (testing "Newly created Card should know a Metabase version used to create it"
+    (let [new-version "just a check"]
+      (t2.with-temp/with-temp [:model/Card card {}]
+        (is (= (:metabase_version card)
+               config/mb-version-string))
```

Comment:
nit: we often write test in this order `(is (= expected result))` it's called the [yoda](https://en.wikipedia.org/wiki/Yoda_conditions) condition.

it's not really important here because `=` is order agnostic, but if you use [=?](https://github.com/metabase/hawk/blob/main/docs/approximately-equal.md), it's required to place `expected` in first.

so I suggest we should keep the convention consistently 

```suggestion
        (is (= config/mb-version-string (:metabase_version card) ))
```